Gemfile

```diff
@@ -1,3 +1,5 @@
 source "https://rubygems.org"
 
 gemspec
+
+gem 'rspec', '~> 3.3.0'
```

Comment:
Why add rspec as a runtime dependency? If rspec must be used, then it should be added to .gemspec as a development dependency.